Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/medical.yml

```diff
@@ -70,6 +71,10 @@
         name: satchel
       - id: CMBackpackMedical
       - id: CMSatchelMedical
+    - name: Mask
+      choices: { CMMask: 1 }
+      entries:
+      - id: CMMaskSterile
```

Comment:
These come included in the room the racks are in with 3 boxes, will be mapped in next update if they aren't already
```suggestion
```